ddl/column.go

```diff
@@ -1443,7 +1444,7 @@ func (w *updateColumnWorker) cleanRowMap() {
 // BackfillData will backfill the table record in a transaction. A lock corresponds to a rowKey if the value of rowKey is changed.
 func (w *updateColumnWorker) BackfillData(handleRange reorgBackfillTask) (taskCtx backfillTaskContext, errInTxn error) {
 	oprStartTime := time.Now()
-	ctx := kv.WithInternalSourceType(context.Background(), w.jobContext.ddlJobSourceType())
+	ctx := kv.WithInternalSourceAndTaskType(context.Background(), w.jobContext.ddlJobSourceType(), kvutil.ExplicitTypeDDL)
```

Comment:
Why it cannot use `WithInternalSourceType`? it should also be ddl?

Reply:
The `ddlJobSourceType()` returns something like "ddl_add_index", so in our current check logic, this can't be recognized as `ddl`.